support-frontend/app/services/pricing/DefaultPromotionsService.scala

```diff
@@ -0,0 +1,69 @@
+package services.pricing
+
+import com.amazonaws.services.s3.AmazonS3URI
+import com.gu.aws.AwsS3Client
+import com.gu.support.config.{Stage, TouchPointEnvironments}
+import io.circe._
+import io.circe.parser._
+import io.circe.generic.auto._
+import akka.actor.ActorSystem
+import com.gu.support.catalog.{GuardianWeekly, Product}
+import com.typesafe.scalalogging.LazyLogging
+import services.pricing.DefaultPromotionsService.DefaultPromotions
+
+import java.util.concurrent.atomic.AtomicReference
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+import scala.util.Try
+
+trait DefaultPromotionsService {
+  def getPromos(product: Product): List[String]
```

Comment:
```suggestion
  def getPromoCodes(product: Product): List[String]
```
To make explicit that we are talking about promo *codes* here not the full promotion object